docs/aws-config.md

```diff
@@ -12,7 +12,11 @@ If you do not have the `aws` CLI locally installed, follow the [user guide][5] t
 
 ## Download Velero
 
-1. Download the [latest release's](https://github.com/heptio/velero/releases) tarball for your client platform.
+1. Download the [latest official release's](https://github.com/heptio/velero/releases) tarball for your client platform.
+
+_We strongly recommend that you use an [official release](https://github.com/heptio/velero/releases) of
```

Comment:
This is messing up the numbering (https://github.com/heptio/velero/blob/9610812d7add8933a957b2e532e8dc2bd583a890/docs/aws-config.md) -- probably need to indent it. same on other pages